### PR TITLE
Adding service for log_power.py

### PR DESCRIPTION
**Pull request by fschlueter**
@cozzyd thankful for any comment

### CODE REVIEW

File: client/log_power.service [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Take and copy monitoring data of the 2024 power system
+
+[Service]
+User=rno-g
+Restart=always
+ExecStart=python3 /home/rno-g/micromini-firmware/client/log_power.py
+ExecReload=/bin/kill -USR1 $MAINPID
```
Comment (cozzyd) [inline client/log_power.service:8]: do you do anything when you get a SIGUSR1? There's no configuration to reread or anything... I would just omit that line!

File: client/Makefile [hunk 2]
```diff
@@ -12,4 +12,6 @@ micromini-tool.completions: micromini-tool
 micromini-tool: micromini-tool.c
 	$(CC) -Wall -o $@ -g -Os $^
 
-
+service-install:
```
Comment (cozzyd) [inline client/Makefile:15]: make micromini-tool a prereq for this?

File: client/log_power.service [hunk 3]
```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Take and copy monitoring data of the 2024 power system
+
+[Service]
+User=rno-g
+Restart=always
+ExecStart=python3 /home/rno-g/micromini-firmware/client/log_power.py
```
Comment (cozzyd) [inline client/log_power.service]: I can't remember if you need the full path to python3 here

Comment (cozzyd) [reply client/log_power.service]: oh oops you already fixed that

File: client/log_power.service (hunk 1 above)
Comment (fschlueter) [reply client/log_power.service:8]: Okay, this was copy past from one of yours.

File: client/Makefile (hunk 2 above)
Comment (fschlueter) [reply client/Makefile]: like this?

Comment (fschlueter) [reply client/Makefile]: Also added the service to all.

Comment (cozzyd) [reply client/Makefile]: yep! that way you can't service-install without compiling the executable

**Review by cozzyd — APPROVED** [review]

**Comment by fschlueter** [comment]
nice. I am testing the changes over night on 12 and will enrol it to all if successful

**Pull request merged**